### ISSUE TITLE
Installation problem on mac os x maverick

### ISSUE THREAD

**Issue by hedget** [peterbraden/node-opencv#134]
hi folks,

when installing node-open cv I had an error:
```
npm ERR! opencv@0.5.0 preinstall: `node-gyp clean rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the opencv@0.5.0 preinstall script.
```
What could be the problem?

Thanks,
HT.

**Comment by SlightlyCyborg** [comment]
<pre><code>
Warning: Already tapped!
Warning: opencv-2.4.10.1 already installed
</code></pre>

**Comment by SlightlyCyborg** [comment]
I installed opencv using <code>brew install opencv --32-bit</code> and that fixed the problem

I noticed that the 32 bit version forced me to install cmake and package config which the 64 did not force upon me. <code>npm install opencv</code> used CXX which is associated with cmake.

After uninstalling 32 bit opencv and reinstalling 64 bit, npm install open cv worked

TLDR; brew install opencv does not install cmake.

<code>brew install cmake</code> will fix your problem :+1:
